SwaglyricsBot/general_commands.py

```diff
@@ -52,7 +51,18 @@ async def get_lyrics_command(self, ctx, song=None, artists=None):
         """
         log = Log()
 
+        def send_lyrics():

```

Comment:
```suggestion
        async def send_lyrics():
```

Reply:
You can't use the await statement in the non-async function, you need to mark `send_lyrics()` as async

Reply:
And ofc you'll need to await each `send_lyrics()` call